src/world/Spell/Spell.cpp

```diff
@@ -0,0 +1,96 @@
+/*
+Copyright (c) 2014-2018 AscEmu Team <http://www.ascemu.org>
+This file is released under the MIT license. See README-MIT for more information.
+*/
+
+#include "Spell.h"
+#include "SpellAuras.h"
+#include "Definitions/AuraStates.h"
+
+SpellCastResult Spell::canCast(bool tolerate)
+{
+    if (p_caster != nullptr && !GetSpellInfo()->IsPassive() && !m_triggeredSpell)
+    {
+        // You can't cast other spells if you have the player flag preventing cast
+        if (p_caster->hasPlayerFlags(PLAYER_FLAG_PREVENT_SPELL_CAST))
+        {
```

Comment:
For this condition (and all similar single line returns) please omit the braces.

Reply:
I think it was mentioned in the wiki under codestyle to not omit braces for single line if-statements 😃 I can be mistaken though.

Reply:
@Appled It's entirely possible, the code style hasn't been updated in a while.